tsconfig.json

```diff
@@ -1,19 +1,11 @@
 {
+    "extends": "tsconfigs/browser-module",
     "compilerOptions": {
+        "esModuleInterop": false,
+        "strictFunctionTypes": false,
         "target": "ES5",
-        "noImplicitAny": true,
-        "sourceMap": true,
-        "strictNullChecks": true,
-        "declaration": true,
-        "removeComments": false,
-        "noUnusedLocals": true,
         "jsx": "react",
-        "jsxFactory": "jsx",
-        "lib": [
-            "dom",
-            "es5",
-            "es2015.core"
-        ]
```

Comment:
[tsconfigs sets this to `["ESNext", "DOM"]`](https://github.com/mightyiam/tsconfigs/blob/1be54f2c1b2384828ced56327f77419e8a2a37a9/browser-module.json#L4). That means that TypeScript won't error if we accidentally attempt to use some environment feature that is newer. But our CI browser tests will. And besides, we should know. Although I've been caught once or twice using a totally new `Object.entries` on a project that targets older environments.